app/controllers/issue_templates_controller.rb

```diff
@@ -110,7 +107,7 @@ def set_pulldown
                                                   @project.id, @tracker.id, true, true).first
 
     unless project_default_template.blank?
-       @default_template = project_default_template
+       default_template = project_default_template.id

```

Comment:
__[rubocop]__
* Use 2 (not 3) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)